application/src/main/kotlin/net/corda/v5/application/messaging/FlowSessionUtils.kt

```diff
@@ -1,37 +0,0 @@
-@file:JvmName("FlowSessionUtils")
-package net.corda.v5.application.messaging
-
-import net.corda.v5.base.annotations.Suspendable
-
-/**
- * Serializes and queues the given [payload] object for sending to the counterparty. Suspends until a response is
- * received, which must be of the given [R] type.
- *
- * Note that this function is not just a simple send+receive pair: it is more efficient and more correct to use this
- * when you expect to do a message swap than do use [FlowSession.send] and then [FlowSession.receive] in turn.
- *
- * @param R The data type received from the counterparty.
- * @param payload The data sent to the counterparty.
- *
- * @return The received data [R]
- */
-@Suspendable
-inline fun <reified R : Any> FlowSession.sendAndReceive(payload: Any): R {
```

Comment:
Ouch that is going to hurt!

Reply:
Yes. Kotlin developer will now feel Java developers' pain.